sbag/generators/javascript/__init__.py

```diff
@@ -56,9 +71,19 @@ def get_path_for_methods(endpoint):
 
 def get_path_parameters(endpoint):
     ret = ''
-    for param in endpoint.parameters:
-        ret = ret + ' + ' + param + ' + ' + "'/'"
+    if '{' in endpoint.path and '}' in endpoint.path:
+        for param in endpoint.parameters:
+            ret = ret + ' + ' + param + ' + ' + "'/'"
+        return ret
+    else:
+        return ret
```

Comment:
Since you return `ret` in both cases, you can take it out of if, and remove else statement.

Reply:
Good point!